### PR TITLE
Use 4.4.0 which is the latest stable Crate version

### PR DESCRIPTION
**Pull request by SStorm**
## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Review by turbo-ele — APPROVED** [review]
❤️

**Comment by SStorm** [comment]
Cannot merge this until https://github.com/docker-library/official-images/pull/9500 is merged, because 4.4.0 is not yet available.

**Review by lukasbals — APPROVED** [review]
Lovely

File: tests/test_bootstrap.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ async def test_bootstrap_license(
                         "secretKeyRef": {"key": "license", "name": f"license-{name}"},
                     },
                     "name": "my-crate-cluster",
-                    "version": "4.3.3",
+                    "version": CRATE_VERSION,
```
Comment (turbo-ele) [inline tests/test_bootstrap.py:112]: 🥇

**Review by turbo-ele — APPROVED** [review]
great!

**Pull request merged**